.github/workflows/continuous-integration.yml

```diff
@@ -76,6 +80,18 @@ jobs:
           registry: ${{ env.DOCKER_REGISTRY }}
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Run rename script if in the template repo
+        run: |
+          if [[ "$GITHUB_REPOSITORY" == */template ]]; then
+            scripts/rename
+          fi
```

Comment:
Sorry, can you talk me through this? If the name of the repo is `*/template`, doesn't that mean it _doesn't_ need to be renamed, since it's not a (e.g.) dataset package (which would be named `*/threedep` or whatever)?

Reply:
It's not strictly needed, but if the rename script fails catastrophically we'll catch it on PRs (or merges into `main` if the PR is from a fork).
It also generates and pushes (almost entirely useless) docker images. Running the rename script results in them having `template` contents rather than `ephemeral` ones. Not particularly useful, but might be handy to validate that it's generating valid projects.

Reply:
Gotcha, in that case I'm :+1: thanks!